Assets/OperatorUserInterface/Widgets/Scripts/Utils/WidgetInteraction.cs

```diff
@@ -20,7 +20,16 @@ public class WidgetInteraction : Singleton<WidgetInteraction>
         public bool allowDwellTime;
 
         public static bool settingsAreActive;
-        
+	public bool showExplanation = true;
+	void Update(){
+		if(Input.GetKeyDown(KeyCode.Escape) && showExplanation == false)
+	{
+			showExplanation = true;	
+	}	else if(Input.GetKeyDown(KeyCode.Escape) && showExplanation == true)
+	{
+			showExplanation = false;	
+	}
```

Comment:
You can simplify this code by writing
```suggestion
	if(Input.GetKeyDown(KeyCode.Escape))
	{
			showExplanation = !showExplanation;	
	}	
```